host_core/lib/host_core/web_assembly/imports.ex

```diff
@@ -16,14 +16,6 @@ defmodule HostCore.WebAssembly.Imports do
   @chunk_threshold 900 * 1024
   @chunk_rpc_timeout 15000
 
-  def fake_wasi(_agent) do
-    %{
-      fd_write:
-        {:fn, [:i32, :i32, :i32, :i32], [:i32],
-         fn _context, _a, _b, _c, _d -> suppress_fdwrite() end}
-    }
-  end
-
```

Comment:
Don't we need this for WASI support?

Reply:
nope. this was for when we had fake wasi support. now we have real WASI support from the engine.